cmd/bootstrap/README.md

```diff
@@ -147,3 +147,15 @@ go run -tags relic ./cmd/bootstrap finalize \
    - root Quorum Certificate for `ClusterBlockProposal` for collector cluster with ID `<ClusterID>`
    - REQUIRED at NODE START by all collectors of the respective cluster
    - file can be made accessible to all nodes at boot up (or recovery after crash)
+
+## Generating networking key for Observer
+
+This generates the networking key used by observers to connect to the public libp2p network. It is a different key format than staked nodes and should only be used for observers.
+
+```bash
+go run -tags relic ./cmd/bootstrap observer-network-key  -o ./path/
+```
+
+In this example, the following file will be generated: `./path/network-key`
```

Comment:
Instead of hardcoding a file name, might be better to just let the user specify the entire file name instead of just the path.

Reply:
yea. that's better. I thought the outputDir command was required, but looks like it's not. so I'll add a new flag for the file path